### DISCUSSION TITLE
DNSWizard and DNS Entity Manager Support

### DISCUSSION THREAD

**Discussion by devhty** [polymind-inc/acmebot#327] [Q&A] [answered]
Hi all, does the keyvault-acmebot certificate renewal solution work with DNSWizard and DNS Entity Manager? Otherwise is there a workaround for this?

**Comment by shibayan** [comment, marked as answer]
I don't know what service the DNS Wizard is, but it is not a service supported by Acmebot, so there will be no workaround until the extension model currently being worked on in PR #286 is incorporated.

**Reply by devhty** [reply]
Thanks for the fast response - is there a timeline as to when PR #286 will be completed?

**Reply by shibayan** [reply]
I'm currently in the process of trying to find an API that is easy to use. If you have any suggestions, please let us know.

**Reply by shibayan** [reply]
Custom DNS feature was released.
